05week/spaceTravelToMars.js

```diff
@@ -9,7 +9,30 @@ let jobTypes = {
   programmer: 'Any Ship!'
 };
 
-// Your code here
+class CrewMember {
+  constructor (name, job, specialSkill, ship) {
```

Comment:
Since `ship` it its own method, you don't need to include it in the constructor.

Reply:
same here. new commit and will merge. thanks!